test/e2e/app-dir/app-static/app/variable-revalidate/layout.js

```diff
@@ -3,15 +3,15 @@ import { cache, use } from 'react'
 export default function Layout({ children }) {
   const getData = cache(() =>
     fetch('https://next-data-api-endpoint.vercel.app/api/random?layout', {
-      next: { revalidate: 10 },
+      next: { revalidate: 100 },
```

Comment:
Why was this changed?

Reply:
I noticed that this test was flaky. It seems to be caused by this timeout because the test was running longer than 10s (combined) so the layout cache renewed in a middle of a test and caused an error.

[`ec92452` (#45432)](https://github.com/vercel/next.js/pull/45432/commits/ec92452adbfb10192addac8cc9e80172e5863d4f)

Reply:
Can you link to where you saw the flake? If this revalidation was flakey on this assertion https://github.com/vercel/next.js/blob/06964dacd887bd6831f94d903f5f308590ba4733/test/e2e/app-dir/app-static/app-static.test.ts#L358 the `revalidate: 3` would also be flakey and instead should ensure the revalidate period starts at the beginning of that assertion.

Reply:
This one failed, just locally.

https://github.com/JanKaifer/next.js/blob/ec92452adbfb10192addac8cc9e80172e5863d4f/test/e2e/app-dir/app-static/app-static.test.ts#L395

Reply:
Yup, that's the assertion that you sent.
Is there an elegant way to refresh the cache in the tests?

Reply:
We could add an internal test header that is only honored when `__NEXT_TEST_MODE` env is present that resets the incremental cache which would make testing things like this more stable 🤔

Reply:
:+1:
Since we have `createNextDescribe`, we can easily add an automatic call in `afterEach` or something.